### ISSUE TITLE
Mixing the normal CSVs and the bird CSVs breaks chart grouping

### ISSUE THREAD

**Issue by Hex27** [btovis/btovis.github.io#204]
Steps to reproduce:

- Upload a normal CSV
- Add a panel
- Upload a bird CSV
The default barchart will lose all its plots

**Comment by CGDogan** [comment]
We'll see if this is a duplicate of #199 only after it's resolved
